### ISSUE TITLE
PathID length calculation error

### ISSUE THREAD

**Issue by abiusx** [OWASP/rbac#15]
The following code on line 120 of rbac.php:

```
    // $row_count = jf::SQL ( "SELECT count(*) FROM " . $this->tablePrefix() . $this->type());
    // $separator_count = --$row_count[0]['count(*)'];
    // $total_char_count = $query_char_count + $separator_count;

    // if ((int) $total_char_count >= 1024)
    //     throw new Exception ( "Path exceeds character count limit" );
```

Is invalid, because it counts for all the nodes that exist, and not just the ones that are being used! And always throws exceptions on big enough data sets. Should be fixed.

**Comment by jburns131** [comment]
Related to Issue #11

**Comment by abiusx** [comment]
Apparently SQLite has no limit on group_concat (this needs to be tested).
MySQL has a 1024 character limit, which can be bypassed with this:
SET SESSION group_concat_max_len = 1000000;

**Comment by jburns131** [comment]
This bug is being addressed in Issue #11. Closing this Issue.

**Comment by jburns131** [comment]
Unit Test completed.
